data/sql/updates/pending_db_world/rev_1571086874024573263.sql

```diff
@@ -0,0 +1,338 @@
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 29475;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`)
+VALUES
+(29475,0,0,1,9,0,100,0,0,10,250,250,0,11,54537,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Improved Land Mine - Victim In Range 0-10 Yards - Cast Spell Detonation'),
+(29475,0,1,0,61,0,100,0,0,0,0,0,0,41,500,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - Linked - Despawn'),
+(29475,0,2,0,5,0,100,0,0,0,0,29618,0,33,29618,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - On Kill - Kill Credit'),
+(29475,0,3,0,5,0,100,0,0,0,0,29619,0,33,29618,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - On Kill - Kill Credit');
+
+-- Garm Invader: Set active after respawn; start random waypoint movement after 1-60 seconds;
+-- disable auto attack / combat movement while following the waypoint path
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 29619;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 29619;
+DELETE FROM `smart_scripts` WHERE `source_type` = 9 AND `entryorguid` BETWEEN 2961900 AND 2961904;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`)
+VALUES
+(29619,0,0,1,11,0,100,0,0,0,0,0,0,48,1,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - On Respawn - Set Active'),
+(29619,0,1,2,61,0,100,0,0,0,0,0,0,20,1,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - Linked - Enable Auto Attack'),
+(29619,0,2,0,61,0,100,0,0,0,0,0,0,21,1,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - Linked - Enable Combat Movement'),
+(29619,0,3,4,1,0,100,0,1000,60000,0,0,0,20,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - OOC - Disable Auto Attack'),
+(29619,0,4,5,61,0,100,0,0,0,0,0,0,21,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - Linked - Disable Combat Movement'),
+(29619,0,5,0,61,0,100,0,0,0,0,0,0,88,2961900,2961904,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - Linked - Call Random Action List'),
+(29619,0,6,0,58,0,100,0,0,0,0,0,0,41,1000,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - On Waypoint Path Ended - Despawn After 1 Second'),
+(29619,0,7,0,6,0,100,0,0,0,0,0,0,41,5000,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Garm Invader - On Death - Despawn After 5 Seconds'),
+
```

Comment:
I think we can avoid this empty line, fromt he IDs a coder can guess that the smart_scripts change, do you agree?

Reply:
It's easier to read in my opinion by separating the script block.

Reply:
:ok_hand: